src/bindfs.c

```diff
@@ -1695,6 +1695,8 @@ static void print_usage(const char *progname)
            " --map=user1/user2:...      Let user2 see files of user1 as his own.\n"
            " --map-passwd=<passwdfile>  Load uid mapping from <passwdfile>.\n"
            " --map-group=<groupfile>    Load gid mapping from <groupfile>.\n"
+           " --map-passwd-rev=<passwdfile>  Load reversed uid mapping from <passwdfile>.\n"
+           " --map-group-rev=<groupfile>    Load reversed gid mapping from <groupfile>.\n"
```

Comment:
Let's line these up :)
```suggestion
           " --map-passwd=...           Load uid mapping from passwd-like file.\n"
           " --map-group=...            Load gid mapping from group-like file.\n"
           " --map-passwd-rev=...       Load reversed uid mapping from  passwd-like file.\n"
           " --map-group-rev=...        Load reversed gid mapping from group-like file.\n"
```